src/main/java/DukeException.java

```diff
@@ -0,0 +1,5 @@
+public class DukeException extends Exception {
+    public DukeException() {
+        super();
```

Comment:
You may consider adding a custom message which is easier for user to understand.
Eg. 
```
public DukeException(String message) {
    super(message);
}
```